lib/apiservers/engine/backends/container.go

```diff
@@ -1768,8 +1768,20 @@ func (c *Container) validateContainerLogsConfig(vc *viccontainer.VicContainer, c
 		since = time.Unix(s, n)
 	}
 
+	// TODO(jzt): this should not require an extra call to the portlayer. We should
+	// update container.DataVersion when we hydrate the container cache at VCH startup
+	// see https://github.com/vmware/vic/issues/4194
 	if config.Timestamps {
-		return unsupported("timestamps")
+		// check container DataVersion to make sure it's supported
+		params := containers.NewGetContainerInfoParams()
+		params.SetID(vc.ContainerID)
+		info, err := PortLayerClient().Containers.GetContainerInfo(params)
+		if err != nil {
+			return 0, 0, err
+		}
+		if info.Payload.DataVersion == 0 {
```

Comment:
@hickeng I captured something similar in https://github.com/vmware/vic/issues/4059, can you take a look and provide input/feedback?